provider/datagen/tests/data/cldr/cldr-core/supplemental/units.json

```diff
@@ -0,0 +1,948 @@
+{
+  "supplemental": {
+    "version": {
+      "_unicodeVersion": "15.0.0",
+      "_cldrVersion": "43"
+    },
+    "unitConstants": {
+      "ft_to_m": {
+        "_value": "0.3048"
+      },
+      "ft2_to_m2": {
+        "_value": "ft_to_m*ft_to_m"
+      },
+      "ft3_to_m3": {
+        "_value": "ft_to_m*ft_to_m*ft_to_m"
+      },
```

Comment:
I think this a good idea, because I am planning to have all the math in u32/u32. it make sense to just put all the u32/u32. We can see after starting the math if there will be overflow or not.

by the way, we are having also `u64` in rust.